app/writer/arctic_monkeys.md

```diff
@@ -5,3 +5,4 @@
 SongFacts in the year:
 
 - January: [Fluorescent Adolescent](../song/jan/fluorescent_adolescent.md)
+- February: [Fluorescent Adolescent](../song/jan/fluorescent_adolescent.md)
```

Comment:
You wrote the wrong directory path, it should be "../song/**feb**/fluorescent_adolescent.md" instead.